Dockerfile

```diff
@@ -4,7 +4,8 @@ ARG GO_VERSION=1.22
 ARG XX_VERSION=1.4.0
 
 # for testing
-ARG DOCKER_VERSION=27.0.3
+ARG DOCKER_VERSION=27.1.0
+ARG DOCKER_CLI_VERSION=${DOCKER_CLI_VERSION-${DOCKER_VERSION}}
```

Comment:
nit: this is just `ARG DOCKER_CLI_VERSION=$DOCKER_VERSION`

Reply:
Oh! You're right; not sure what I was thinking there. Let me update; and also update to v27.1.1